src/LightningDB.Tests/EnvironmentTests.cs

```diff
@@ -145,7 +145,7 @@ public void CanOpenEnvironmentMoreThan50Mb()
         }
         
         
-#if (NETCOREAPP3_1 || NET5_0)
+#if NETCOREAPP3_1 || NET5_0 || NET6_0
```

Comment:
#if NETCOREAPP3_1_OR_GREATER

Though I am confused since these are the only targets, so maybe unnecessary?